schema/url/pass/plugin-twobytwo.raml

```diff
@@ -3,23 +3,52 @@
 types:
   # Type name determines the name of the Java class that will be generated.
 
+  TwoByTwoSpec:
+    type: MosaicSpec
+    properties:
+      xAxisReferenceValue: string
+      yAxisReferenceValue: string
+
+  TwoByTwoPostRequest:
+    type: VisualizationRequestBase
+    properties:
+      config: TwoByTwoSpec
+
+  # TODO once we get the other stats tables on this path we can add df, configuration notes, error msg etc
+  Statistic:
+    additionalProperties: false
+    properties:
+      value: number
+      confidenceInterval?: string # (min - max)
+      confidenceLevel?: number
+      pvalue: string # number to two signifs or string '<0.0001'
```

Comment:
Or NA?

Reply:
That's a good question. Right now an NA in plot.data gets written to Json as a null. So it's a question of whether we prefer that or a string 'NA' for these I guess..

Reply:
i would think null would be easier to handle in the frontend, instead of checking for if that string = NA. But perhaps we already do this elsewhere and have a better handling of it than i know.